README.md

```diff
@@ -154,24 +146,25 @@ It can be used as such:
     let parentDir = "path/to/directory";
     
     // leave second parameter empty to use first path as directory
-    checker(parentDir, null, null, "exampleCheckDir", false, true, false)
+    checker(parentDir, null, "exampleCheckDir", null, false, false, true)
         .then( /*...*/ );
         
     // in this example, no files will be written, and Debug loggers are silenced
 ```
 
 
+__Note:__ The erc-checker will automatically remove all temporary files on termination. To prevent this, set an environment variable `DEV=true`.
 
 
 #### Debug
 
-To receive command line outputs from the erc-checker's node module, please set the environment variable DEBUG first.  
-E.g. if your project uses the module, start it as such:
+Enable debugging by setting an environment variable **DEBUG**.
+E.g.:
  ```bash
     $  DEBUG=* node yourProject.js
```

Comment:
Use the actual file name here instead of `yourProject.js`, i.e. `index.js` (or `checker.js`??)